app/scripts/views/mixins/experiment-mixin.js

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'lib/experiments/open-gmail',
+  'lib/experiments/mailcheck',
+  'lib/experiments/coppa-view',
+  'lib/experiments/null',
+  'lib/url'
+], function (OpenGmailExperiment, MailcheckExperiment, CoppaExperiment, NullExperiment, Url) {
+  'use strict';
+
+  var VERIFICATION_TYPE = 'verification';
+  var VERIFICATION_ABLE = 'verificationExperiment';
+  var FORCE_EXPERIMENT_PARAM = 'forceVerificationExperiment';
+  // available verification experiments that must be independent.
+  var VERIFICATION_EXPERIMENTS = {
```

Comment:
This name is confusing, only `openGmail` is on the `go verify your email` page, the other two are on the signup page.